src/Pact/Repl/Lib.hs

```diff
@@ -144,6 +146,14 @@ replDefs = ("Repl",
      "Output VALUE to terminal as unquoted, unescaped text."
      ,defZRNative "env-hash" envHash (funType tTyString [("hash",tTyString)])
      "Set current transaction hash. HASH must be a valid BLAKE2b 512-bit hash. `(env-hash (hash \"hello\"))`"
+     ,defZNative "test-capability" testCapability
+      (funType tvA [("capability", TyFun $ funType' tTyBool [])]) $
+     "Allow the testing of the scope of a capability CAPABILITY in the repl environment. This is intended for testing " <>
+     "ONLY, and one should revoke all capabilities after by calling 'revoke-all-capabilities' after use." <>
+     "`$(grant-capability (TRANSFER) (my-module.transfer sender receiver 1.0))`"
```

Comment:
Suggest more "operational" tone for comment to match other caps docs. EDIT also let's get rid of `revoke-all-capabilities` as they are automatically removed with tx management as described below:
> Specify and request grant of CAPABILITY. Once granted CAPABILITY and any composed capabilities are in scope for the rest of the transaction. Allows direct invocation of capabilities which is not available in the blockchain environment.

Reply:
Also should return tTyString

Reply:
Fixed. I've also added the test to confirm that `commit-tx` revokes.